scripts/set-dnsmasq

```diff
@@ -6,7 +6,12 @@
 
 # prep
 set -e
+# if the config file doesn't exist, use the default config
 DNSMASQ_CONF_PATH="./docker/dnsmasq/dnsmasq.conf"
+if [ ! -f "$DNSMASQ_CONF_PATH" ] || [ ! -s "$DNSMASQ_CONF_PATH" ]; then
+  cp ./docker/dnsmasq/dnsmasq.default.conf "$DNSMASQ_CONF_PATH"
+fi
+
```

Comment:
Alternatively (probably better), dnsmasq supports more than one config file:

https://serverfault.com/questions/1156182/is-there-anyway-to-include-an-external-rules-file-other-than-hosts-in-dnsmasq-co

So we can all use the base config, and sndev can write to another .gitignore file I think.